engine/baml-runtime/src/internal/llm_client/primitive/openai/types.rs

```diff
@@ -6,6 +6,212 @@ use serde::{
 pub type CompletionResponse = ChatCompletionGeneric<CompletionChoice>;
 pub type ChatCompletionResponse = ChatCompletionGeneric<ChatCompletionChoice>;
 
+/// OpenAI Responses API response structure
```

Comment:
advice, keep only the bare minimum of what we need here. otherwise if they change the API we will break due to failure to parse. so the best thing we can do is ignore all data that we don't need instead of maintaining a 1:1

Reply:
ah makes sense!
resolved via https://github.com/BoundaryML/baml/pull/2103/commits/49f284c336db436ac669e168d47b260c2f772572